### PR TITLE
Remove warnings regarding duplicate late declaration of packages

### PR DESCRIPTION
**Pull request by mugdhadhole1**
ISSUE: #72

### CODE REVIEW

File: trlc/parser.py [hunk 1]
```diff
@@ -1835,16 +1835,6 @@ def parse_preamble(self, kind):
             self.stab.register(self.mh, pkg)
         else:
             pkg = self.stab.lookup(self.mh, self.ct, ast.Package)
-            if pkg.declared_late and kind == "trlc":
-                # lobster-trace: LRM.Duplicate_Late_Packages
```
Comment (christophkloeffel) [inline trlc/parser.py:1839]: this is a trace to the LRM
please have a look at the LRM and also remove the linked content in the LRM

**Review by christophkloeffel — CHANGES_REQUESTED** [review]
Please also create an entry in https://github.com/bmw-software-engineering/trlc/blob/main/CHANGELOG.md

**Review by christophkloeffel — CHANGES_REQUESTED** [review]
there is a tracing file in "rbt-late-package-declarations" test folder. make sure to delete this, because you deleted the corresponding entry in the LRM

**Review by akashsurwase1 — APPROVED** [review]
ok to merge

**Review by christophkloeffel — APPROVED** [review]

**Pull request merged**